rs/nns/governance/src/proposals/manage_network_economics_tests.rs

```diff
@@ -0,0 +1,135 @@
+use super::*;
+use crate::proposals::self_describing::LocallyDescribableProposalAction;
+use ic_nns_governance_api::SelfDescribingValue;
+use maplit::hashmap;
+
+#[test]
+fn test_network_economics_all_fields() {
+    let network_economics = NetworkEconomics::with_default_values();
+
+    let self_describing_value =
+        SelfDescribingValue::from(network_economics.to_self_describing_value());
+
+    assert_eq!(
+        self_describing_value,
+        SelfDescribingValue::Map(hashmap! {
+            "reject_cost_e8s".to_string() =>
+                SelfDescribingValue::Nat(candid::Nat::from(100_000_000u64)),
```

Comment:
```suggestion
                SelfDescribingValue::from(100_000_000u64),
```

Out of scope; later PR, maybe.